GameData/AirplanePlus/Compatibility/FAR.cfg

```diff
@@ -387,39 +387,39 @@
 }
 
 //remove lifting values for some non wing parts like cockpits
-@PART[144cockpit]:FOR[FerramAerospaceResearch]
+@PART[144cockpit]:NEEDS[FerramAerospaceResearch]
```

Comment:
Never use :FOR when referencing a different mod, or MM will always apply the patch; always use :NEEDS instead.